src/main/java/seedu/address/logic/commands/person/ViewGroupCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.commands.person;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.IsTagInPersonPredicate;
+
+/**
+ * Command that lists all contacts who share the same tag
+ */
+public class ViewGroupCommand extends Command {
+
+    public static final String COMMAND_WORD = "viewGroup";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": List out all contacts that "
+            + "share the same tag. "
+            + COMMAND_WORD;
```

Comment:
Can an example here, and give parameters similar to the MESSAGE_USAGE in AddCommand.java

Reply:
Alright